Dockerfile

```diff
@@ -1,55 +1,29 @@
-FROM ubuntu:18.04
-LABEL maintainer="Mirage"
-ENV PY_VER "3.7.4"
-ENV PY_VER_MAJOR "3.7"
+FROM heroku/heroku:20
```

Comment:
Probably due to review this dev container

Reply:
Concur!!